src/discussions/post-comments/comments/CommentsSort.jsx

```diff
@@ -21,16 +22,31 @@ function CommentSortDropdown({
   const sortedOrder = useSelector(selectCommentSortOrder);
   const [isOpen, open, close] = useToggle(false);
   const [target, setTarget] = useState(null);
-
   const handleActions = (reverseOrder) => {
     close();
     dispatch(setCommentSortOrder(reverseOrder));
   };
 
+  const enableCommentsSortTour = useCallback((enabled) => {
+    const data = {
+      enabled,
+      tourName: 'response_sort',
+    };
+    dispatch(updateUserDiscussionsTourByName(data));
+  });
```

Comment:
it will create on every render which will lose its purpose. use like this
const enableCommentsSortTour = useCallback((enabled) => {
    const data = {
      enabled,
      tourName: 'response_sort',
    };
    dispatch(updateUserDiscussionsTourByName(data));
  },[]);